### PR TITLE
String localization

### PR DESCRIPTION
**Pull request by louiscoste**

### CODE REVIEW

File: src/components/add-awards-dialog.js [hunk 1]
```diff
@@ -96,8 +96,8 @@ class AddAwardsDialog extends BaseMixin(LitElement) {
 					${this.availableAwards.map(award => this._renderAward(award))}
 				</d2l-list>
 			</div>
-			<d2l-button slot='footer' primary data-dialog-action=${DONE_ACTION}>Done</d2l-button>
-			<d2l-button slot='footer' data-dialog-action=${CANCEL_ACTION}>Cancel</d2l-button>
+			<d2l-button slot='footer' primary data-dialog-action=${DONE_ACTION}>${this.localize('done-action')}</d2l-button>
```
Comment (quinnbudan) [inline src/components/add-awards-dialog.js:99]: The `title-text` attribute in the `d2l-dialog` should probably also be localized

**Comment by BraydonKains** [comment]
image

File: src/components/awards-classlist-dialogs.js [hunk 2]
```diff
@@ -284,7 +284,7 @@ class AwardsClasslistRevokeDialog extends BaseMixin(LitElement) {
 			>
 			<label>
 				<span class="d2l-input-label">
-					Select Award *
+					${this.localize('issue-awards-dialog-select-label')} *
```
Comment (quinnbudan) [inline src/components/awards-classlist-dialogs.js:287]: Here as well the `title-text` attribute should be localized I think

**Comment by quinnbudan** [comment]
The tab names in `awards.js` should also probably be localized

**Comment by louiscoste** [comment]
@BraydonKains 
image

**Pull request merged**